lms/djangoapps/grades/new/course_grade.py

```diff
@@ -24,8 +26,35 @@ class CourseGrade(object):
     def __init__(self, student, course, course_structure):
         self.student = student
         self.course = course
+        self.course_version = getattr(course, 'course_version', None)
+        self.course_edited_timestamp = getattr(course, 'subtree_edited_on', None)
         self.course_structure = course_structure
         self.chapter_grades = []
+        self.persisted_grade = None
```

Comment:
Is there much value in storing this as a dict instead of storing the fields individually?